tests/workers/test_ecs_worker.py

```diff
@@ -522,15 +522,23 @@ async def test_launch_types(
 @pytest.mark.parametrize(
     "cpu,memory", [(None, None), (1024, None), (None, 2048), (2048, 4096)]
 )
+@pytest.mark.parametrize("container_cpu,container_memory", [(1024, 2048)])
```

Comment:
If we expect that `cpu` and `memory` will be inherited from the task to the container, we should add cases to the test matrix where the container cpu and memory are empty and verify that